browser-extension/src/addTournesolRecommendations.js

```diff
@@ -257,3 +265,40 @@ function loadRecommandations() {
     handleResponse
   );
 }
+
+function millifyViews(videoViews) {
+  // Intl.NumberFormat object is in-built and enables language-sensitive number formatting
+  return Intl.NumberFormat('en', { notation: 'compact' }).format(videoViews);
+}
+
+function viewPublishedDate(publishedDate) {
+  const date1 = new Date(publishedDate);
+  const date2 = new Date();
+  // we will find the difference in time of today's date and the published date, and will convert it into Days
+  // after calculating no. of days, we classify it into days, weeks, months, years, etc.
+  const diffTime = date2.getTime() - date1.getTime();
+  const diffDays = Math.floor(Math.abs(diffTime / (1000 * 3600 * 24)));
+
+
+  if (diffDays == 0) {
+    return `Today`;
+  } else if (diffDays < 31) {
+    if (diffDays < 14) {
+      return `${diffDays} days ago`;
+    } else {
+      return `${Math.floor(diffDays / 7)} weeks ago`;
+    }
+  } else if (diffDays >= 31 && diffDays < 365) {
```

Comment:
The `diffDays >= 31` part is not needed and adds repetition.

Reply:
I applied this constraint in order to differentiate singular and plural in terms of month

Reply:
I said it's not needed because we are in the `else` of an `if (diffDays < 31)`, so we are already certain `diffDays` is `>= 31`.

Reply:
Got it, thanks!